content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -448,16 +448,18 @@ A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; the
 
 #### DNS
 
-A ServiceAccount for `kube-dns` is created in the `kube-system` namespace.
+A ServiceAccount for `coredns` is created in the `kube-system` namespace.
 
 Deploy the kube-dns Deployment and Service:
 
```

Comment:
@neolit123 The CoreDNS service is named `kube-dns`. 
Should we mention something like:  "The CoreDNS service is named `kube-dns`. This is done to prevent any interruption in service when the user is switching the cluster DNS from kube-dns to CoreDNS or vice-versa."
@johnbelamaric what do you think?

Reply:
yikes..well it's confusing to say the least.
any plans to phase this out eventually?

Reply:
cc @timothysc

Reply:
@neolit123 the naming has been kept on purpose. This prevents service loss when upgrading the cluster DNS.

Reply:
@rajansandeep ok, i will make an update, including the note possibly somewhere near the top of the page.